ge07_team04/models/motorcycle_registry.py

```diff
@@ -0,0 +1,24 @@
+from odoo import api, models, fields
+from odoo.exceptions import ValidationError
+
+class MotorcycleRegistry(models.Model):
+
+    _inherit = 'motorcycle.registry'
+
+    stock_lot_ids = fields.One2many(comodel_name="stock.lot", inverse_name="motorcycle_registry_id", string="Lot IDs")
+    stock_lot_id = fields.Many2one(comodel_name="stock.lot", compute='_compute_lot_id', string="Lot ID", store=True)
+    sale_order_id = fields.Many2one(comodel_name='sale.order')
+    
+    vin = fields.Char(string="VIN", related="stock_lot_id.name", required=False)
+    owner_id = fields.Many2one(comodel_name="res.partner", string="Owner", related="sale_order_id.partner_id")
+
+    @api.constrains('stock_lot_ids')
+    def _check_lot_ids(self):
+        lots = [x for x in self if x.stock_lot_ids]
+        if len(lots) > 1: raise ValidationError('A registered motorcycle can only belong to one lot.')
```

Comment:
When adding  raise Error(Validation or user) always be sure to add the option to translate the message. By default string in views and fields are added to the export translation file, but messages in error don't, so you need to add something like this: 
<img width="913" alt="Captura de pantalla 2023-07-27 a la(s) 15 05 37" src="https://github.com/bneb-odoo/team4-kawiil-motors/assets/107058475/2c9798e2-7125-4d3a-bdb9-113d840061d0">


Pd: Don't forget import the '_ '